### ISSUE TITLE
lots of false detection results on background

### ISSUE THREAD

**Issue by ShiyangZhang** [lyxok1/Tiny-DSOD#21]
Hi， guys，
I used tiny-dsod to train a idcard-detection model. It perferms well. But for pictures without idcard it falsely detected some "idcard".

The training data cotained pictures each has at least one idcard inside.

we tried to train the model on pure background dataset which with no idcard rect. the loss is always zero. but we test the model, it also falsely detected some "idcard".

Can you give me any sugguestion？
